tests/test_Source/test_source.py

```diff
@@ -141,6 +141,36 @@ def setup_method(self):
                 "dec_off",
             ),
         )
+
+        self.source_dict5 = Table(
```

Comment:
@ahuang314, you added this source dict but have not used it. I think you can remove this dict.

Reply:
Oops, I meant to use that for source6, but since it looks like it is not needed, I removed it